src/utils/api.js

```diff
@@ -20,4 +20,14 @@ export const specificArticle= (article_id) => {
         .catch((error)=>{
             return error
         })
+}
+
+export const commentSection= (article_id) => {
+    return api.get(`/articles/${article_id}/comments`)
+        .then((response)=>{
+            return response.data
+        })
+        .catch((error)=>{
+            return error
```

Comment:
Having this could cause an issue. As it stops your error being thrown and makes it into a value that will go to a .then block rather than a .catch